src/bartholomew.rs

```diff
@@ -42,6 +43,24 @@ pub fn render(req: Request) -> Result<Response> {
 
     // Load the site config.
     let mut config: template::SiteInfo = toml::from_slice(&std::fs::read(CONFIG_FILE)?)?;
+    let dynamic_templates = config
+        .dynamic_templates
+        .clone()
+        .unwrap_or_default()
+        .iter()
```

Comment:
Since you've cloned, I _think_ you could to an `into_iter()` here and retain ownership of the elements, so you wouldn't need to `to_owned` them in the map.  No promises though!

Reply:
Thanks! fixed that.